server/utils/checkYourAnswersUtils.ts

```diff
@@ -243,7 +243,7 @@ export const getApplicantDetails = (application: Application | Cas2SubmittedAppl
         text: 'Nationality',
       },
       value: {
-        html: nationality,
+        html: nationality || 'Unknown',
```

Comment:
Should we add a unit test for this? I went looking but couldn't find this `getApplicantDetails` method tested at all in `checkYourAnswersUtils.test.ts`, so I suspect we may be able to create a prior commit that adds tests for the previous behaviour?